src/static.d.ts

```diff
@@ -25,54 +25,70 @@ export const enum TransferType {
   NO_TRANSFER_POSSIBLE,
 }
 
+export const enum LocationType {
+  STOP = 0,
+  STATION,
+  ENTRANCE_EXIST,
+  GENERIC_NODE,
+  BOARDING_AREA
+}
+
 // files
 
 export interface Agency {
-  agency_id: string;
+  agency_id?: string;
   agency_name: string;
   agency_url: string;
   agency_timezone: string;
-  agency_lang: string;
-  agency_phone: string;
-  agency_fare_url: string;
+  agency_lang?: string;
+  agency_phone?: string;
+  agency_fare_url?: string;
+  agency_email?: string;
 }
 
 export interface Stop {
   stop_id: string;
-  stop_code: string;
-  stop_name: string;
-  stop_desc: string;
-  stop_lat: number;
-  stop_lon: number;
-  zone_id: string;
-  stop_url: string;
-  location_type: 0 | 1;
-  parent_station: string;
-  stop_timezone: string;
+  stop_code?: string;
+  stop_name?: string;
+  stop_desc?: string;
+  stop_lat?: number;
+  stop_lon?: number;
+  zone_id?: string;
+  stop_url?: string;
+  location_type?: LocationType;
+  parent_station?: string;
+  stop_timezone?: string;
+  wheelchair_boarding?: 0 | 1 | 2;
```

Comment:
ideally we would use an enum here too, maybe: `{ UNKNOWN=0, ACCESSIBLE, NOT_ACCESSIBLE }`

Reply:
Done